tests/mocha.opts

```diff
@@ -1,2 +1,3 @@
 --colors
+--compilers js:pouchdb-plugin-helper/mocha-babel 
```

Comment:
Alternatively I would be okay with just converting from ES modules to CommonJS and using only Node 4+ features. We only support Node 4+ anyhow. Then we could get rid of the `mocha-babel` helper entirely.